plugins/uefi-capsule/efi/sbat.csv.in

```diff
@@ -0,0 +1 @@
+fwupd,1,Fedora,0,34,0
```

Comment:
doesn't this need some `@`'s?

Reply:
That's a good question. I'm not sure to be honest.

I thought about using at least `@PACKAGE_NAME@` instead of hardcoding `fwupd` but that's set `conf.set_quoted('PACKAGE_NAME', meson.project_name())` and so is enclosed with quotes which I don't think is what we want.

The other fields are the following from the SBAT spec:

```
SBAT_DateVersion           // version of this structure 1 at initial release
ComponentName; 	      	  // for example grub2
ComponentGeneration;       // 1 at initial release then incrementing
ProductName;   	      	  // for example: Oracle Linux
ProductGeneration;     	  // generally 0 unless	needed
ProductVersion;	      	  // for example: "7.9"
VersionGeneration;     	  // generally 0 unless needed
```
but none of these can be taken from templating macros AFAICT. Maybe we could add some of these to `meson.build` so it can be set by downstream users / distros ? I'm not that familiar with meson to know what's the proper way to do this configurable.

Reply:
Maybe I'm asking a stupid question; can't we get all the OS data from /etc/os-release in the python script?

Reply:
That actually makes a lot of sense to me, given that most distros ship that file.

I'll explore that approach and maybe also not even including a dummy csv in the repo but generating a dynamic one at build time using `/etc/os-release` and a set of meson configuration data.

But probably I'll do that on Monday since I've to work on other things today.

Reply:
> That actually makes a lot of sense to me, given that most distros ship that file.
> 
> I'll explore that approach and maybe also not even including a dummy csv in the repo but generating a dynamic one at build time using `/etc/os-release` and a set of meson configuration data.
> 
> But probably I'll do that on Monday since I've to work on other things today.

That sounds preferable to me to this approach.